aten/src/ATen/gen.py

```diff
@@ -217,13 +242,30 @@ def format_yaml(data):
     return yaml.dump(data, default_flow_style=False, Dumper=noalias_dumper, width=float('Inf'))
 
 
-def generate_storage_type_and_tensor(backend, density, declarations):
+def add_op_registrations(per_type_registrations, per_op_registrations, op_registrations):
+    for op_registration in op_registrations:
+        opname = op_registration.operator_name
+        registration = op_registration.registration_code
+        # apply whitelist
+        if op_registration_whitelist and opname not in op_registration_whitelist:
```

Comment:
nit: then here `if op_registration_whitelist is not None and ...`

Reply:
done